api_functional_v4_test.go

```diff
@@ -801,7 +801,7 @@ func TestRemovePartiallyUploaded(t *testing.T) {
 	if err == nil {
 		t.Fatal("Error: PutObject should fail.")
 	}
-	if err.Error() != "Proactively closed to be verified later." {
+	if err.Error() != "proactively closed to be verified later" {
```

Comment:
Test was failing without this change, 'proactively closed to be verified later' message should be the same as returned by the go routine created just above.